### PR TITLE
image apm increase upload size for PHP

### PR DESCRIPTION
**Pull request by Song2017**
image apm increase upload size for PHP

### CODE REVIEW

File: docker-library/apache-php-mysql/php.ini [hunk 1]
```diff
@@ -4,4 +4,7 @@ error_log=/var/log/httpd/php-error.log
 display_errors=Off
 display_startup_errors=Off
 date.timezone=UTC
-
+; increase upload size for PHP
+post_max_size=20M
+upload_max_filesize=20M
+memory_limit=128M
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/php.ini]: why do we have a limit for memory to 128M?

File: docker-library/apache-php-mysql/php.ini [hunk 2]
```diff
@@ -4,4 +4,7 @@ error_log=/var/log/httpd/php-error.log
 display_errors=Off
 display_startup_errors=Off
 date.timezone=UTC
-
+; increase upload size for PHP
+post_max_size=20M
+upload_max_filesize=20M
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/php.ini]: what's difference between them?

File: docker-library/apache-php-mysql/php.ini (hunk 1 above)
Comment (Song2017) [reply docker-library/apache-php-mysql/php.ini]: 128M is default max memory value memory_limit. 8M before PHP5.2.0, 16M in PHP 5.2.0

File: docker-library/apache-php-mysql/php.ini (hunk 2 above)
Comment (Song2017) [reply docker-library/apache-php-mysql/php.ini]: In briefly, post_max_size = upload_max_filesize * file counts. 
Upload_max_filesize set limit for the upload file but post_max_size is checked for all kind of POST data.
In according to customer requirement, the import database file should be one.

File: docker-library/apache-php-mysql/php.ini (hunk 1 above)
Comment (Song2017) [reply docker-library/apache-php-mysql/php.ini]: here is direct url: [https://stackoverflow.com/questions/11305516/how-to-import-large-sql-file-in-phpmyadmin ](https://stackoverflow.com/questions/11305516/how-to-import-large-sql-file-in-phpmyadmin )

**Review by fanjeffrey — APPROVED** [review]

**Pull request merged**